src/match.py

```diff
@@ -1,5 +1,6 @@
 class Match:
-    def __init__(self, stadium, date, host, guest, status=False, attendance=None, events=None, statistics=None):
+    def __init__(self, number, stadium, date, host, guest, status=False, attendance=None, events=None, statistics=None):
```

Comment:
It seems the match _number_ is not needed.
Let it be for now, it's okay, I saw that you already implemented part of the logic with it.
Then maybe we can change it.